xmtp_mls/src/subscriptions/stream_messages/test_case_builder.rs

```diff
@@ -298,6 +299,20 @@ impl StreamSequenceBuilder {
     }
 
     fn init_session(&mut self, groups: Vec<GroupTestCase>) {
+        // Set up db mock expectation
+        let db_calls = || {
+            let mut mock_db = xmtp_db::mock::MockDbQuery::new();
+            // Set up expectations for commonly called db methods
+            mock_db
+                .expect_get_latest_sequence_id_for_group()
+                .returning(|_| Ok(None));
+            mock_db
```

Comment:
Do the tests fail if we don't set this expectation? I can't find anywhere we're calling just the latest sequence id